pkg/vsphere/disk/vmdk.go

```diff
@@ -58,7 +64,34 @@ func LockedVMDKFilter(vm *mo.VirtualMachine) bool {
 	return vm.Runtime.PowerState == types.VirtualMachinePowerStatePoweredOn
 }
 
-func IsLockedError(err error) bool {
-	// TODO: add check here for actual error
-	return true
+func IsLockedError(op trace.Operation, err error) bool {
```

Comment:
This is another option, for the logic in these two methods will be similar, and not that easy to consolidate.
But I'm Okay with it, as it will release the dependency of my PR and this one.